lib/transactional.pm

```diff
@@ -157,3 +160,16 @@ sub trup_shell {
     process_reboot 1 if $args{reboot};
 }
 
+# When transactional-update is triggered manually is required to wait for rollback.service
+# to not be running. This is needed because rollback.service is triggered on first boot
+# after updates or rollbacks.
+# The transactional-update.timer waits for that service to be finished before starting itself.
+# In general automated services will make sure that they don't block each other,
+# but this does not apply when manual triggering of the script.
+sub ensure_rollback_service_not_running {
+    for (1 .. 12) {
+        # last if (script_run("systemctl show -p SubState --value rollback.service") !~ 'running|start');
+        my $output = script_output("systemctl show -p SubState --value rollback.service");
+        $output =~ '^(dead|exited)$' ? last : sleep 10;
```

Comment:
Hmm, method name states ensure, but in case service is not `dead/exited` we will do more iterations, but it won't fail test in case.

Reply:
good catch! I would like to try `$output =~ '^(start|running)$' ? sleep 10 : last;` to be sure that those two are the states that we need, but there are issues at the moment in OSD to run verification.

Reply:
It worked, now the test took less time to run: https://openqa.suse.de/tests/4105892
good that you noticed.